README.md

```diff
@@ -28,10 +34,14 @@ if err != nil {
 
 cfg.ConnConfig.Tracer = otelpgx.NewTracer()
 
-conn, err := pgxpool.NewWithConfig(ctx, cfg)
+conn, err := pgxpool.NewConfig(ctx, cfg)
```

Comment:
Why this has been changed? I don't see `NewConfig` function in `github.com/jackc/pgx/v5/pgxpool`